scripts/taxonomy/GetTaxonomyTable.pl

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env perl
+# Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+# Copyright [2016-2019] EMBL-European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+use strict;
+use warnings;
+use Getopt::Long;
+use Cwd qw(abs_path);
+use File::Spec::Functions qw(catfile);
+use Bio::EnsEMBL::Compara::DBSQL::DBAdaptor;
+
+$| = 1;
+
+# Parameters:
+my $help = 0;
+# URL to the Compara database
+my $compara_url;
+# Output directory where the taxonomy table will be saved
+my $out_dir = Cwd::abs_path();
+# Handle input arguments
+GetOptions("help"          => \$help,
+           "compara_url=s" => \$compara_url,
+           "outdir=s"      => \$out_dir
```

Comment:
The simplest is not to have any option, just print to the standard output and people can redirect the output wherever they want with `>` or even pipe it into something.
If there is an option in this script, I would expect it to be the path to the output file, i.e. `-o /path/to/dest/file`. This way, the script can be run with its output in a given file without needing a shell.
I find having an option to change the output directory unusual. Imagine the script is going to be run across several compara databases (vertebrates, plants, fungi): you need to run it in three different directories, otherwise the output files will be overwritten